src/shared/varlink-io.systemd.service.c

```diff
@@ -27,3 +27,40 @@ int varlink_method_ping(Varlink *link, JsonVariant *parameters, VarlinkMethodFla
 
         return varlink_reply(link, NULL);
 }
+
+int varlink_method_set_log_level(Varlink *link, JsonVariant *parameters, VarlinkMethodFlags flags, void *userdata) {
+        static const JsonDispatch dispatch_table[] = {
+                {"level", JSON_VARIANT_INTEGER, json_dispatch_int64, JSON_MANDATORY},
```

Comment:
uh, 4th column here is the offset actually, you passed the flag. need to add `, 0`